packages/components/src/components/hds/accordion/item/index.ts

```diff
@@ -45,6 +58,62 @@ export default class HdsAccordionItemComponent extends Component<HdsAccordionIte
     return this.args.containsInteractive ?? false;
   }
 
+  /**
+   * @param toggleTextSize
+   * @type {HdsTextSizes}
+   * @default 'medium'
+   */
+  get toggleTextSize() {
+    const size = this.args.size ?? DEFAULT_SIZE;
+    const sizeMap = {
+      large: 300,
+      medium: 200,
+      small: 100,
```

Comment:
Super, ultra nit: For HdsAccordionSizeValues below, sizes are ordered: small, medium, large (ie in reverse order).

Reply:
Added mixed nested examples in 8bade6c